kernel_crawler/deb.py

```diff
@@ -74,8 +74,8 @@ def scan_packages(cls, stream):
                 if key in ('Provides', 'Depends'):
                     value = value.split(', ')
             except ValueError:
-                print(line)
-                raise
+                # Just skip the line if it is malformed
+                continue
```

Comment:
Malformed lines can happen; in my tests, a line `Description:` was breaking this. Just skip it.